8720 allow metadata re export in smaller batches (#8721)
IQSS/dataverse: doc/sphinx-guides/source/admin/metadataexport.rst

Batch exports through the API 
-----------------------------

> In addition to the automated exports, a Dataverse installation admin can start a batch job through the API. The following 4 API calls are provided: 

Reviewer: ```suggestion
In addition to the automated exports, a Dataverse installation admin can start a batch job through the API. The following four API calls are provided: 
```

``curl http://localhost:8080/api/admin/metadata/exportAll``

``curl http://localhost:8080/api/admin/metadata/reExportAll``

``curl http://localhost:8080/api/admin/metadata/clearExportTimestamps``

``curl http://localhost:8080/api/admin/metadata/reExportDataset?persistentId=doi:10.5072/FK2/AAA000``

The first will attempt to export all the published, local (non-harvested) datasets that haven't been exported yet. 
The second will *force* a re-export of every published, local dataset, regardless of whether it has already been exported or not. 

> The first two calls return a status message informing the administrator, that the process has been launched (``{"status":"WORKFLOW_IN_PROGRESS"}``). The administrator can check the progress of the process via log files: ``[Payara directory]/glassfish/domains/domain1/logs/export_[time stamp].log``.

Reviewer: ```suggestion
The first two calls return a status message informing the administrator that the process has been launched (``{"status":"WORKFLOW_IN_PROGRESS"}``). The administrator can check the progress of the process via log files: ``[Payara directory]/glassfish/domains/domain1/logs/export_[time stamp].log``.
```

> Instead of running the 'reExportAll' the same can be accomplished using 'clearExportTimestamps' followed by 'exportAll'. 

Reviewer: ```suggestion
Instead of running "reExportAll" the same can be accomplished using "clearExportTimestamps" followed by "exportAll".
```

> The difference is that when the exporting prematurely fails due to some problem, the datasets that did not get exported yet still have the timestamps cleared. A next call to exportAll will skip the datasets already exported and try to export the ones that still need it. 

Reviewer: ```suggestion
The difference is that when exporting prematurely fails due to some problem, the datasets that did not get exported yet still have the timestamps cleared. A next call to exportAll will skip the datasets already exported and try to export the ones that still need it. 
```

> Calling clearExportTimestamps should return *{"status":"OK","data":{"message":"cleared: X"}}* where 'X' is the total number of datasets. 

Reviewer: ```suggestion
Calling clearExportTimestamps should return ``{"status":"OK","data":{"message":"cleared: X"}}`` where "X" is the total number of datasets cleared.
```

> The reExportDataset call gives you the opportunity to *force* a re-export only a specific dataset and with some script automation could allow you the export specific batches of datasets. This might be usefull when handling exporting problems or when reExportAll takes too much time and is not really needed. . 

Reviewer: ```suggestion
The reExportDataset call gives you the opportunity to *force* a re-export of only a specific dataset and (with some script automation) could allow you the export specific batches of datasets. This might be usefull when handling exporting problems or when reExportAll takes too much time and is overkill.
```
